src/App.tsx

```diff
@@ -1,3 +1,4 @@
+import '@walletconnect/react-native-compat'
```

Comment:
```suggestion
// Needs to be imported as early as possible for the polyfills to be there
import '@walletconnect/react-native-compat'
```

Reply:
Yeah I was thinking about that as well but given the order of imports, feels weird to have a comment on the very first line as this would imply a comment about the whole file.

But yeah may just move it down a notch and add the comment 🙏

Reply:
78d0846